lib/redis-manager.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable global-require */
+/* eslint-disable import/no-dynamic-require */
+
+'use strict';
+
+const { promisify } = require('util');
+const md5 = require('md5');
+const path = require('path');
+const logger = require('@janiscommerce/logger');
+const redis = require('redis');
+const CacheManagerError = require('./cache-manager-error');
+
+/**
+*    RedisManager class
+*/
+class RedisManager {
+
+	get MS() {
+		return process.env.MICROSERVICE || 'node';
+	}
+
+	/**
+     * Get the Redis config
+     * @return {object}
+     */
+	get config() {
+		if(!this._config)
+			this.cacheConfig();
+
+		return this._config;
+	}
+
+	set keyPrefix(prefix) {
+		this._keyPrefix = prefix;
+	}
+
+	get keyPrefix() {
+		return this._keyPrefix;
+	}
+
+	getKey(key) {
+		return `${this.keyPrefix}${key}`;
+	}
+
+	/**
+     * Get the Redis Config JSON path
+	 * @returns {String} route path
+     */
+	get configPath() {
+		return path.join(process.cwd(), 'config/redis.json');
+	}
+
+	/**
+     * Cache the Redis config.
+	* @param {String} route Config JSON path
+     */
+	cacheConfig() {
```

Comment:
```suggestion
	setConfig() {
```
Este metodo guarda la configuracion de redis para devolverla en el getter config, por lo que el nombre del método seria mas descriptivo si fuera `setConfig` en vez de `cacheConfig`